src/App.js

```diff
@@ -0,0 +1,46 @@
+import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import User from './components/Authentication/User';
+import { setCurrentUser } from './Redux/UserReducer';
+import DoctorsList from './components/Doctors/DoctorsList';
+import DrDetail from './components/DoctorDetail/DrDetail';
+import 'react-toastify/dist/ReactToastify.css';
+import ProtectRoutes from './components/ProtectRoutes';
+import Appointments from './components/BookAppointments/Appointments';
+import Display from './components/DisplayAppointments/Display';
+import NewDoctor from './components/Doctors/NewDoctor';
+import Navbar from './components/Navbar/Navbar';
+import About from './components/About';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

Reply:
We have refactored the code